### PR TITLE
Fix broken antora component links to docs

### PR DESCRIPTION
**Pull request by mmattel**
Fixes the references to documents in docs, eg release notes.

Backport to 10.9 and 10.8

### CODE REVIEW

File: modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc [hunk 1]
```diff
@@ -92,7 +92,7 @@ ownCloud provides two encryption types:
 | *User-Key:*
 a| Every user has their own private/public key pairs, and the private key is protected by the user’s password.
 
-IMPORTANT: User-Key encryption has been depreciated with xref:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
+IMPORTANT: User-Key encryption has been depreciated with xref:{latest-docs-version}@docs:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc:95]:
```suggestion
IMPORTANT: User-Key encryption has been deprecated with xref:{latest-docs-version}@docs:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
```

We could also fix the word depreciated => deprecated while we are here, if it does not cause anything else to explode.

File: modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc [hunk 2]
```diff
@@ -248,7 +248,7 @@ You may only disable encryption by using the xref:configuration/server/occ_comma
 
 === Limitations of User-Key-Based Encryption 
 
-* Depreciated with xref:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
+* Depreciated with xref:{latest-docs-version}@docs:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc:251]:
```suggestion
* Deprecated with xref:{latest-docs-version}@docs:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
```

File: modules/admin_manual/pages/configuration/files/encryption/encryption_configuration_quick_guide.adoc [hunk 3]
```diff
@@ -62,7 +62,7 @@ If the master key has been compromised or exposed, you can replace it. You will
 
 == User-Key-Based Encryption
 
-IMPORTANT: User-Key encryption has been depreciated with xref:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
+IMPORTANT: User-Key encryption has been depreciated with xref:{latest-docs-version}@docs:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/encryption/encryption_configuration_quick_guide.adoc:65]:
```suggestion
IMPORTANT: User-Key encryption has been deprecated with xref:{latest-docs-version}@docs:ROOT:server_release_notes.adoc#deprecation-note-for-user-key-storage-encryption[ownCloud release 10.7]
```

File: site.yml [hunk 4]
```diff
@@ -8,6 +8,11 @@ content:
   - url: .
     branches:
     - HEAD
+  # the docs git is just present for the test build to satisfy used refrences
```
Comment (phil-davis) [inline site.yml:11]:
```suggestion
  # the docs git is just present for the test build to satisfy used references
```

**Comment by mmattel** [comment]
@phil-davis many thanks for all the reviews 😃

**Review by phil-davis — APPROVED** [review]

**Pull request merged**